### PR TITLE
ToolCard and Explorer page listing styled

### PR DESCRIPTION
**Pull request by sevtechcodes**

### CODE REVIEW

File: app/(auth)/login/page.tsx
Comment (Wyna-7) [inline app/(auth)/login/page.tsx:1]: I think we are not using this file anymore, but (auth)/sign-in/[[sign-in]]/page.tsx

File: app/(auth)/register/page.tsx
Comment (Wyna-7) [inline app/(auth)/register/page.tsx:1]: I think we are not using this file anymore, but (auth)/sign-up/[[sign-up]]/page.tsx

File: app/components/ToolCard.tsx
Comment (Wyna-7) [inline app/components/ToolCard.tsx:1]: I thought I was working on the toolCard, there  mayb have been a miscommunication 😂
let's see both versions in a call later, and pick the best of  both sides

File: tailwind.config.ts [hunk 1]
```diff
@@ -13,8 +13,8 @@ const config = {
       center: true,
       padding: '2rem',
       screens: {
-        '2xl': '1400px',
-        sm: '430px',
+        // '2xl': '1400px',
```
Comment (Wyna-7) [inline tailwind.config.ts:16]: can we  remove these if they are not needed anymore?

**Review by Wyna-7 — CHANGES_REQUESTED** [review]

**Review by Wyna-7 — APPROVED** [review]

**Review by jemimacecil — APPROVED** [review]

**Pull request merged**